Include/internal/pycore_code.h

```diff
@@ -285,110 +285,56 @@ PyAPI_FUNC(PyObject*) _Py_GetSpecializationStats(void);
 #define EVAL_CALL_STAT_INC_IF_FUNCTION(name, callable) ((void)0)
 #endif  // !Py_STATS
 
-// Cache values are only valid in memory, so use native endianness.
-#ifdef WORDS_BIGENDIAN
+// NOTE: These cache reading/writing utilities use memcpy to avoid voilating C's
```

Comment:
Could you rewrite this comment bearing in mind that the reader won't have seen the previous version.
A comment shouldn't justify the change. It should explain why the code is the way it is.

Maybe something like:

> Utility functions for reading and writing 32 bit and 64 bit values to and from the inline cache.
We use memcpy to let the C compiler handle unaligned accesses and endianness issues.
memcpy also seems to produce better code than manual copying for most compilers.